scripts/SANS/sans/gui_logic/models/RowEntries.py

```diff
@@ -68,7 +69,7 @@ def __eq__(self, other):
                 self.can_direct_period,
                 self.can_scatter_period,
                 self.can_transmission_period,
-                self.sample_direct,
+                self.sample_direct_period,
```

Comment:
Was this previously not working?

Reply:
well, `self.sample_direct` is in the comparison tuple twice, and `self.sample_direct_period` isn't in there so I think this is just a mistake. There is a test which tests a deep copy, it failed when I removed the `__eq__` override but passes with this correction.

Reply:
Ah I understand now, thanks for explaining!